python/plugins/db_manager/db_plugins/postgis/connector.py

```diff
@@ -181,7 +181,7 @@ def fetchone(self):
         self._execute()
         if len(self.result) - self.cursor:
             res = self.result[self.cursor]
-            ++self.cursor
+            self.cursor += 1
```

Comment:
Nice catch ...
I wonder if flake8 or other tools could have noticed this kind of mistake.

Thanks

Reply:
The bad thing is that it went unnoticed because (I think) it was executed inside a QgsTask, otherwise you would have got the usual Python error dialog.

Reply:
I just tried in my default python interpreter and QGIS, no Python error: 

```python
>>> i = 0
>>> ++i
0
>>>i
0
```

Reply:
Yeah, right. It's not a syntax error, it just does nothing :)